languages/python/oso/tests/test_repr.py

```diff
@@ -20,7 +20,7 @@ def test_repr_when_logging(polar, capsys):
     polar.load_str("f(_foo: Foo) if 1 = 1;")
     list(polar.query_rule("f", Foo()))
     captured = capsys.readouterr()
-    assert f"QUERY RULE: f({FOO_REPR})" in captured.out
+    assert f"QUERY RULE: f({FOO_REPR} TYPE `Foo`)" in captured.out
```

Comment:
I think it makes sense to print `Foo` instead of `{FOO_REPR}` here since `__repr__` is overridden at the instance level